### PR TITLE
Bug 1343805 - Use a darker font for comments, and also avoid Courier

### PR DESCRIPTION
**Pull request by SebastinSanty**

### CODE REVIEW

**Comment by dylanwh** [comment]
Per recent comment on bug, s/Fira Code/Fira Mono/g

File: extensions/BugModal/web/bug_modal.css [hunk 1]
```diff
@@ -598,7 +598,7 @@ h3.change-name {
 
 .comment-text {
     white-space: pre-wrap;
-    font: 13px/1.2 "Droid Sans Mono",Menlo,Monaco,"Courier New",Courier,monospace;
+    font: 13px/1.2 "Droid Sans Mono",Menlo,Monaco,"Fira Code",Consolas,monospace;
```
Comment (floatingatoll) [inline extensions/BugModal/web/bug_modal.css]: There are ligature concerns with "Fira Code" in the bug; can we use a simple monospace variant of Fira instead?

File: skins/contrib/Mozilla/global.css [hunk 2]
```diff
@@ -337,7 +337,7 @@ hr {
 #bugzilla-body {
     background: none;
     border: none;
-    color: #404040;
+    color: #202020;
```
Comment (floatingatoll) [inline skins/contrib/Mozilla/global.css]: Technically, we could just use color: #000 here, but if the goal is to *intentionally* lower the contrast of the text vs. the white background, then it makes sense that we're up in the #222/444 ranges.

File: extensions/BugModal/web/bug_modal.css (hunk 1 above)
Comment (floatingatoll) [inline extensions/BugModal/web/bug_modal.css]: Fira Mono, apparently

Comment (SebastinSanty) [inline extensions/BugModal/web/bug_modal.css]: Yes, that would be better. I'll update it.

File: skins/contrib/Mozilla/global.css (hunk 2 above)
Comment (SebastinSanty) [inline skins/contrib/Mozilla/global.css]: I am not sure about this, as it would affect the whole BMO. I would like to defer this to @dylanwh

**Comment by dylanwh** [comment]
Closing this because the font of the bike shed is apparently very important.

**Pull request closed without merging**